api/onboardingpb/tips.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package onboardingpb;
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "inventorypb/nodes.proto";
+import "inventorypb/services.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "api/onboardingpb;onboardingpb";
+
+// Request message for GetOnboardingStatus method.
+message GetOnboardingStatusRequest {
+  // The User ID.
+  int32 userId = 1;
+}
+
+message TipModel {
+  // The ID of the tip to retrieve.
+  int32 tipId = 1;
+  // Show whether tip is completed or not.
+  bool isCompleted = 2;
+}
+
+// Response message for GetOnboardingStatusResponse method.
+message GetOnboardingStatusResponse {
+  // shows system tips which can be completed once per PMM instance
+  repeated TipModel systemTips = 1;
+  // shows system tips which can be completed once per user
+  repeated TipModel userTips = 2;
+}
+
+// Request message for CompleteUserTip method.
+message CompleteUserTipRequest {
+  // The ID of the tip to retrieve.
+  int32 tipId = 1;
+  // user id which completes user tip.
+  int32 userId = 2;
+}
+
+// Response message for CompleteUserTip method.
+message CompleteUserTipResponse {}
+
+// Service service provides public methods for managing Services.
+service TipService {
+  // GetTipStatus returns info about tip status
+  rpc GetOnboardingStatus(GetOnboardingStatusRequest) returns (GetOnboardingStatusResponse) {
+    option (google.api.http) = {get: "/v1/onboarding/{userId}"};
```

Comment:
this endpoint to be called by authorized user, we will have userId in the context.
we don't need to provide API with userId

Reply:
I don't know how to retrieve it. If you take a look how other endpoints implemented - they do the same, path user id via request param or body

https://github.com/percona/pmm/blob/8ba03139901af250e2180f006a8c55651daaf6a3/api/userpb/user.proto#L20